### PR TITLE
[Functions] (chore): update to solidity v0.8.19

### PR DESCRIPTION
**Pull request by justinkaseman**

### CODE REVIEW

**Review by RensR — APPROVED** [review]

File: contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol
Comment (KuphJr) [inline contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol:1]: This syntax for maps is SOOOO much nicer :)

File: contracts/src/v0.8/functions/dev/1_0_0/ocr/OCR2Base.sol
Comment (KuphJr) [inline contracts/src/v0.8/functions/dev/1_0_0/ocr/OCR2Base.sol:1]: Missing pragma here?

**Review by KuphJr — COMMENTED** [review]
Maybe missing pragma in one file? Re-tag me for approval when addressed.

**Review by KuphJr — COMMENTED** [review]
Maybe missing pragma in one file? Re-tag me for approval when addressed.

File: contracts/src/v0.8/functions/dev/1_0_0/ocr/OCR2Base.sol
Comment (RensR) [reply contracts/src/v0.8/functions/dev/1_0_0/ocr/OCR2Base.sol:1]: OCR2Base should be fine at ^0.8.0 I think

**Pull request merged**